src/components/Books.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import MyBook from './BookItem';
+import Form from './BookForm';
+
+function Books() {
+  const Books = [
+    {
+      author: 'Norman Vincent',
+      title: 'The Power of positive Thinking',
+    },
+    {
+      author: 'Queen Juma',
+      title: 'Fashion Stars',
+    },
+    // {
+    //   author: 'Prince Caleb',
+    //   title: 'The power of Love',
+    // },
```

Comment:
**Optional**

- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.